src/wp-admin/js/inline-edit-post.js

```diff
@@ -91,7 +134,12 @@ inlineEditPost = {
 		$('table.widefat tbody').prepend( $('#bulk-edit') ).prepend('<tr class="hidden"></tr>');
 		$('#bulk-edit').addClass('inline-editor').show();
 
+		// Get the selected posts based on the checked, checkboxes in the post table.
+		// Create a HTML div with the title and a link(delete-icon) for each selected post.
 		$( 'tbody th.check-column input[type="checkbox"]' ).each( function() {
+			// If the checkbox in row for a post is selected,
```

Comment:
This entire comment is unclear. Also, considering it's a multiline comment, use block comments instead of single line comments.